R/CellGraphAssay.R

```diff
@@ -115,6 +115,14 @@ CreateCellGraphAssay <- function (
   ...
 ) {
 
+  # Unlist dots
+  dots <- list(...)
+  if ("command" %in% names(dots)) {
+    command <- dots[["command"]]
+  } else {
+    command = "CreateCellGraphAssay"
+  }
+
```

Comment:
Could you explain what is going on here?

Reply:
Is it related to `arrow` specifically?

Reply:
Perhaps `sys.call()` could be used to access which command was used?

Reply:
This is perhaps unnecessary, but the idea is to log what operation triggered the edgelist copy. This is appended to the global option `pixelatorR.edgelist_copies` when the function is run. At any time, users can check the command list with `getOption("pixelatorR.edgelist_copies")` which returns a tibble with the  command that was run, the path to the edgelist directory that was created and and a time stamp.

Reply:
For example, if I start a new session and run:
````
pxl_file <- system.file("extdata/PBMC_10_cells",
  "Sample01_test.pxl",
  package = "pixelatorR"
)
seur_obj <- ReadMPX_Seurat(pxl_file)
seur_subset <- subset(seur_obj, cells = colnames(seur_obj)[1:5])
````
I can then run `getOption("pixelatorR.edgelist_copies")` to get:
<img width="635" alt="Screenshot 2024-03-18 at 15 59 07" src="https://github.com/PixelgenTechnologies/pixelatorR/assets/23736938/a97ffb8b-6401-41fd-af91-b30ac4a78d1f">